### PR TITLE
Fix Login Form

### PR DESCRIPTION
**Pull request by eudisd**
We know this works because of all the test written in https://github.com/VIDY/embed.js/pull/14 😉

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -266,7 +268,7 @@ export default function Vidy(opts) {
 			(player = helpers.create('div')).className = 'vidy-player'; // position
 			(tmp = helpers.create('div')).className = 'vidy-mask circle'; // size
 			tmp.onclick = () => {
-				if (login.classList.contains('show')) {
+				if (login && login.classList && login.classList.contains('show')) {
```
Comment (eudisd) [inline src/index.js]: Need all these `login` checks.  The page errors out otherwise

**Comment by eudisd** [comment]
Forgot to add the input/name suggestions you asked for, will push that up now

**Comment by eudisd** [comment]
Actually on second thought, I'm going to add this `name` update in the other PR, so I can test it.

File: src/index.js [hunk 2]
```diff
@@ -388,14 +390,17 @@ export default function Vidy(opts) {
...
+
 					// Login submit handler
 					login.onsubmit = e => {
 						e.preventDefault();
-						let x = login.elements;
+						const formElements = login.elements;
 						post('auth/login', {
-							email: x[0].value, // TODO: this is bad
-							password: x[1].value, // TODO: also, validation
+							email: formElements[1].value, // TODO: this is bad
```
[the remark this answers is not in the document]
Comment (eudisd) [reply src/index.js]: Not sure, when I did local testing with the old stuff, x[0] did not exist and x[1] contained the email, so this was residue from the older implementation.  I'm going to use the name props as you suggest add tests for this.

File: src/index.js (hunk 1 above)
Comment (eudisd) [reply src/index.js]: This is me just being extra safe 😉 

I'll remove the extra check on classList

**Comment by eudisd** [comment]
I pushed up the classList removal.  Let me know if there is anything else

**Comment by eudisd** [comment]
@lukeed let me know if you see any more issues.  I'll merge right after

File: src/index.js [hunk 3]
```diff
@@ -388,14 +389,17 @@ export default function Vidy(opts) {
 					tmp.src = DEFAULT_AVATAR;
 				} else {
 					tmp.src = DEFAULT_AVATAR;
-					helpers.append(vidy, (login = helpers.form()));
+
+					const login = helpers.form();
```
Comment (lukeed) [inline src/index.js]: Why the `const` here? A `login` is already declared up above: https://github.com/VIDY/embed.js/pull/16/files#diff-1fdf421c05c1140f6d71444ea2b27638R29

**Pull request merged**